MAINTAINERS.md

```diff
@@ -1,5 +1,97 @@
+* Merging pull requests from contributors
+* Resolving conflict found on pull request reviews
+* Proactively dealing with technical debt
+* Supporting documentation
+* Guiding the projects strategic direction
+
+In other words, maintainers are involved in day-to-day operations of the
+project.
+
+## OPI Maintainer Prerequisites
+
+You should understand the [contributing](https://github.com/opiproject/opi/blob/main/CONTRIBUTING.md)
+guidelines if you want to become a maintainer.
+
+## OPI Maintainer Grant/Revocation Policy
+
+An OPI maintainer is a participant in OPI who has the ability to merge pull
+requests directly into main branch on OPI repositories. This ability grants
+the maintainers the ability to affect the progress of the OPI project in a way
+non-maintainers cannot. Being an OPI maintainer represents a significant level
+of trust to the broader project. Maintainer rights are not granted lightly,
+and will be reviewed as necessary to ensure that trust has not been violated.
+
+### Granting Maintainer Rights
+
+Granting maintainer rights will be considered when a candidate has demonstrated
+the following in their interation with the project:
+
+* Contribution of significant new features to OPI.
+  * Submissions are free of obvious defects.
+  * Submissions do not require many revisions.
+* Consistent and meaningful code reviews of other people's pull requests.
+* Helping others in the community via channels such as the mailing list,
+  GitHub. Issues and PRs, and the Slack channels
+* The individual should have plans for sustained contribution to the project.
+
+The process to grant someone the maintainer role is simple:
+
+* An existing maintainer nominates someone by sending a message to the
+  maintainers slack channel.
+* Existing maintainers discuss the idea of giving the candidate maintainer
+  rights.
+* After discussion has ended, existing maintainers will vote on approving the
+  new maintainer.
+* To recieve maintainership, the candidate must receive a majority of yes votes
+  and zero no votes.
+* One final discussion with the candidate happens to ensure the candidate is
+  ready to accept the maintainer role.
+* If successful, the candidate is added to the appropriate GitHub group to
+  gain maintainer access.
+
+### Revoking Maintainer Rights
+
+When a maintainer acts in a way which is either detrimental to the OPI project,
+it raises a situation which needs to be handled quickly to ensure the long-term
+success and stability of the project. It is possible the maintainer in question
+will have their maintainer rights removed. The process for this is as follows:
+
+* Discuss with the individual privately about the concerns. Use email for this,
+  sticking to the facts, and try to reach a resolution on the individuals.
+  behavior. Ensure the individual understands their behavior needs to change.
+* Send an email discussing the individuals behavior to the maintainer mailing
+  list. Work to resolve the issue in this forum.
```

Comment:
Should add a sentence where the maintainer agrees to have role revoked and process to do that, else move on to steps in line 76 and ff.

Reply:
Good idea!